charts/refinery/values.yaml

```
@@ -133,6 +133,16 @@ environment: []
   #       name: honeycomb
   #       key: api-key
 
+# Configure container lifecycle hooks.
+# This can be used to add a prestop hook to make rolling restarts more robust for instance.
+lifecycle: {}
+  # preStop:
+  #   exec:
+  #     command:
+  #       - "sh"
+  #       - "-c"
+  #       - "sleep 10"
```

```suggestion
lifecycle: {}
```

I'd prefer not to suggest anything here since we use the `scratch` image and can't really run commands.